Assets/Scripts/UI/MainMenu.cs

```diff
@@ -0,0 +1,49 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+
+using ProjectPorcupine.Localization;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class MainMenu : MonoBehaviour
```

Comment:
We are already calling MainMenu to the bottom Menu in the game... should this be the MainScene?